perf-measures/bundle-check/scripts/process-results.ts

```diff
@@ -1,14 +1,23 @@
-import * as fs from 'node:fs/promises'
+import * as fs from 'node:fs'
 
 async function main() {
-    const currentResult = (await fs.readFile('./generated/after.js')).byteLength
-    let previousResult: number | null = null
-    try {
-        previousResult = (await fs.readFile('./generated/before.js')).byteLength
-    } catch (e) {}
-    const table = ['| | Current | Previous |', '| --- | --- | --- |']
-    table.push(`| Bundle Size | ${currentResult} | ${previousResult || 'N/A'} |`)
-    console.log(table.join('\n'))
+    const args = process.argv.slice(2);
+    const filePath = args[0];
+
+    const bundleSize = fs.statSync(filePath).size
+    const metrics = []
+    metrics.push({
+        key: 'bundle-size',
+        name: 'Bundle Size',
+        value: bundleSize,
+        unit: 'B'
```

Comment:
The unit of byte is not converted and is fixed as Byte.
We thought that the number of digits would not change that much, and it would be easier to check the difference without rounding off the units.